### ISSUE TITLE
Unable to see fix messages

### ISSUE THREAD

**Issue by miguelao33** [swri-robotics/mapviz#796]
Hello everyone, thanks a lot for mapviz!

I have an ardusimpe ZED-F9P with this node ([https://github.com/aussierobots/ublox_dgnss]) publishing on /fix topic, which I believe is enough at least to see single points of location on the map. 

As you can see, the frame id of the message is "ubx" and I dont know if it has any influence on that. 

![image](https://github.com/swri-robotics/mapviz/assets/137102087/bfe33187-c731-4493-ac46-e52c57094ae6)

I'm still not running robot_localization_nav_sat_node. tons of covariance (I'm inside the building!)

Any idea of how to represent that data there?

Thanks!

**Comment by danthony06** [comment]
Is it an RTK receiver?

**Comment by miguelao33** [comment]
No, is configured right now as a Rover. I'll buy another ardusimple board configured as RTK soon.

**Comment by danthony06** [comment]
Closing as stale.
